debian/changelog

```diff
@@ -1,3 +1,117 @@
+      glib-networking will also be required.
+    + GDBusMethodInvocation now drops replies if the sender set the
+      NO_REPLY_EXPECTED flag
+    + several GApplication fixes, including fixes for commandline arguments in
+      interpreted languages on Windows
+  * debian/libglib2.0-0.symbols: Update with new symbols for this release.
+  * 0001-regex-test-expect-ASSERTION_EXPECTED-for-ab-with-PCR.patch: Drop,
+    it's included in this release.
+
+ -- Iain Lane <laney@debian.org>  Wed, 20 Jan 2016 17:55:16 +0000
+
+glib2.0 (2.47.4-1) experimental; urgency=medium
+
+  * New upstream release
+    + The GApplication documentation has been improved in several areas.
+  * 0001-tests-fix-a-test-on-32-bit-builds.patch,
+    0001-gtypes.h-move-G_STATIC_ASSERT-to-function-scope.patch: Drop, applied
+    upstream in this release.
+  * 0001-regex-test-expect-ASSERTION_EXPECTED-for-ab-with-PCR.patch: Fix regex
+    tests to assert the right errors as of pcre 8.38. Cherry-pick from
+    upstream. (Closes: #808842)
+  * Don't build automatic dbgsym package for -refdbg
+
+ -- Iain Lane <laney@debian.org>  Thu, 14 Jan 2016 18:27:02 +0000
+
+glib2.0 (2.47.3-3) experimental; urgency=medium
+
+  * debian/patches/0001-gtypes.h-move-G_STATIC_ASSERT-to-function-scope.patch:
+    Another cherry-pick. Should fix g-ir-scanner.
+
+ -- Iain Lane <laney@debian.org>  Sun, 29 Nov 2015 18:45:29 +0000
+
+glib2.0 (2.47.3-2) experimental; urgency=medium
+
+  * debian/patches/0001-tests-fix-a-test-on-32-bit-builds.patch: Cherry-pick
+    from upstream. Fix tests (and therefore the build) on 32 bit arches.
+
+ -- Iain Lane <laney@debian.org>  Thu, 26 Nov 2015 16:12:12 +0000
+
+glib2.0 (2.47.3-1) experimental; urgency=medium
+
+  * New upstream release
+    + New API: hardware-assisted helpers for overflow-checked integer math.
+    + Fixes: Invalid free in g_local_file_trash() (LP: #1512826)
+  * d/p/{Doc-copy-included-example-files.patch,
+    Doc-Fix-missing-glibconfig.h-when-builddir-srcdir.patch,
+    Build-gdbus-example-objectmanager-server-again.patch}: Remove - in this
+    upstream release.
+
+ -- Iain Lane <laney@debian.org>  Wed, 25 Nov 2015 17:59:33 +0000
+
+glib2.0 (2.47.1-1) experimental; urgency=medium
+
+  * New upstream release.
+    + The Unicode support has been updated to version 8.0 of the Unicode standard
+    + GDesktopAppInfo no longer sets the DISPLAY environment variable when
+      launching apps. This is now done in the GAppLaunchContext
+      implementations when appropriate.
+  * debian/watch: Look for development versions too.
+  * debian/patches/90_gio-modules-multiarch-compat.patch: Refresh to apply on
+    this version.
+  * debian/patches/0001-GDateTime-test-fix-occasional-failures.patch: Drop,
+    upstream in this release.
+  * debian/libglib2.0-0.symbols: Update with new symbols for this release.
+
+ -- Iain Lane <laney@debian.org>  Wed, 04 Nov 2015 17:28:23 +0000
+
 glib2.0 (2.46.2-0) master; urgency=medium
```

Comment:
Oh yeah i was just saying remove our old 3.18 `force lower version for jenkins` entry and just keep the latest one of those at the top.

So our changelog looks like debians + our little mangle at the end. Rather than this weird not increasing chain of version numbers.